src/monitoring_service/handlers.py

```diff
@@ -381,16 +381,19 @@ def action_monitoring_triggered_event_handler(event: Event, context: Context) ->
         non_closing_signer=event.non_closing_participant,
     )
     if monitor_request is None:
+        log.error("MonitorRequest was lost or deleted before triggering", monitor_request=monitor_request)
```

Comment:
I'd write something like: "MonitorRequest cannot be found"

Also, it doesn't make sense to log `monitor_request` as it is `None`, but I'd log `token_network_address` and `channel_id`.

Reply:
haha true